runtime/src/bank/builtins/core_bpf_migration/target_core_bpf.rs

```diff
@@ -200,6 +229,7 @@ mod tests {
             })
             .unwrap(),
             &Pubkey::new_unique(), // Not the upgradeable loader
+            false,
```

Comment:
Why false here (and following)? Seems like it confuses things.

Reply:
Program data account isn't executable. I can maybe split these two functions if that's more clear.

Reply:
Oh, I'm sorry, I didn't expand the test case fully enough in github 😞 
Clear enough when the State info is obvious.